### PR TITLE
(maint) merge up 7.x

### PR DESCRIPTION
**Pull request by jonathannewman**
* 7.x:
  (maint) update pyenv to 3.7.13
  (PE-34636) update clj-parent to 5.2.5, bc references
  (maint) update travis integration test refs
  (maint) Update readme build status link
  (docs) update release notes date
@jonathannewman

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -107,15 +107,15 @@ jobs:
       env: PDB_TEST=core+ext/openjdk8/pg-11
       script: *run-core-and-ext-tests
 
-    # === integration with main branches
+    # === integration with 7.x branches
```
Comment (nmburgan) [inline .travis.yml:110]: Probably need to put all this back

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# PuppetDB [![Build Status](https://travis-ci.com/puppetlabs/puppetdb.svg?branch=master)](https://travis-ci.com/puppetlabs/puppetdb)
+# PuppetDB [![Build Status](https://app.travis-ci.com/puppetlabs/puppetdb.svg?branch=7.x)](https://app.travis-ci.com/puppetlabs/puppetdb)
```
Comment (nmburgan) [inline README.md:1]: And make this main?

File: .travis.yml (hunk 1 above)
Comment (jonathannewman) [reply .travis.yml:110]: Good catch.

File: documentation/release_notes_7.markdown [hunk 3]
```diff
@@ -17,7 +17,7 @@ canonical: "/puppetdb/latest/release_notes.html"
 
 ## PuppetDB 7.11.1
 
-Released August 11 2022
+Released September 14 2022
```
Comment (nmburgan) [inline documentation/release_notes_7.markdown:20]: Suppose maybe this doesn't matter, but I'm not sure?

Comment (jonathannewman) [reply documentation/release_notes_7.markdown:20]: These were legit updates.  I think it is ok.

**Review by nmburgan — APPROVED** [review]

**Pull request merged**